src/rax.h

```diff
@@ -203,17 +206,19 @@ extern void *raxNotFound;
 /* Exported API. */
 rax *raxNew(void);
 int raxInsert(rax *rax, unsigned char *s, size_t len, void *data, void **old);
-int raxTryInsert(rax *rax, unsigned char *s, size_t len, void *data, void **old);
+int raxTryInsert(rax *rax, unsigned char *s, size_t len, void *data,
+                 void **old);
```

Comment:
Its OK if lines are >80 lines. Be careful with whitespace changes as it makes it harder to merge upstream Redis code. 

This one should be reverted for sure as it provides no readability benefits on modern monitors and will just make merging difficult.

Reply:
This is an unintentional change...I don't remember doing this. Will fix it for later.